packages/mobile/src/geth/consts.ts

```diff
@@ -2,10 +2,6 @@ export const WEI_PER_CELO = 1000000000000000000.0
 export const UNLOCK_DURATION = 600
 
 export enum GethSyncMode {
-  // Sync mode to imply no local geth node but an infura-like setup.
```

Comment:
Removed as ZeroSync mode is no longer encoded as a "sync mode" - else we have no way to encode which SyncMode to use when geth is turned on